postgres/summary-views.md

```diff
@@ -0,0 +1,24 @@
+```sql
```

Comment:
Note: once these summaries are introduced in #263, they can be removed from here (& their prompt)